lib/constants/error-messages.ts

```diff
@@ -0,0 +1,44 @@
+// Platform Error
+export const PLATFORM_FAILURE = 'platform-failure';
+export const PLATFORM_NOT_FOUND = 'platform-not-found';
+export const BAD_CREDENTIALS = 'bad-credentials';
+export const INTEGRATION_UNAUTHORIZED = 'integration-unauthorized';
+export const RATE_LIMIT_EXCEEDED = 'rate-limit-exceeded';
+export const AUTHENTICATION_ERROR = 'authentication-error';
+export const AUTHENTICATION_FAILURE = 'authentication-failure';
```

Comment:
Looks like we should combine/consolidate/deduplicate these two. I think `AUTHENTICATION_ERROR` is the best to keep.